fem/fespace.cpp

```diff
@@ -3273,4 +3273,888 @@ void QuadratureInterpolator::MultTranspose(
    MFEM_ABORT("this method is not implemented yet");
 }
 
+template<int T_D1D =0, int T_Q1D =0, int T_NBZ =0>
+static void D2QValues2D(const int NE,
+                        const Array<double> &b_,
```

Comment:
Should this go into a different file now?

Reply:
Yes, it is in `fem/quadinterpolator.cpp` now